go/vt/sqlparser/parse_test.go

```diff
@@ -3912,6 +3915,39 @@ func TestCreateTable(t *testing.T) {
 	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on delete cascade on update set default,
 	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on delete set default on update set null,
 	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on delete set null on update restrict,
+	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update set default on delete cascade ,
+	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update set null on delete set default,
+	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update restrict on delete set null,
+	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update no action,
+	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update cascade
+)`,
+			output: `create table t (
```

Comment:
This specific output was needed since the parser parses the actions into a struct so the ordering specifically of this is lost. 

We have verified though that MySQL doesn't accidentally allows a list of these items, just only both orders so there's no potential semantic difference due to the different order.